pkg/services/ngalert/provisioning/persist.go

```diff
@@ -24,3 +24,20 @@ type ProvisioningStore interface {
 type TransactionManager interface {
 	InTransaction(ctx context.Context, work func(ctx context.Context) error) error
 }
+
+type ProvenanceOrgAdapter struct {
```

Comment:
We now had 3 places where this struct was pasted in both impl and tests. Rule of 3 => put it somewhere shared.

I'm not the biggest fan of needing this. Perhaps the provenance store API could be improved to make this not necessary. But, that wasn't the goal of this PR and would require even more files to be touched.